python/lsst/ts/envsensors/device/mock_device.py

```diff
@@ -106,31 +96,105 @@ def _format_temperature(self, i: int):
         s: `str`
             A string representing a temperature.
         """
-        temp = random.uniform(MockTemperatureConfig.min, MockTemperatureConfig.max)
         if i < self._missed_channels:
             return ""
+
+        prefix = f"C{i:02d}="
+        value = random.uniform(MockTemperatureConfig.min, MockTemperatureConfig.max)
         if i == self._disconnected_channel:
-            return f"C{i:02d}={DISCONNECTED_VALUE}"
-        return f"C{i:02d}={temp:09.4f}"
+            value = float(DISCONNECTED_VALUE)
+        return f"{prefix}{value:09.4f}"
+
+    def _format_hx85a(self, i: int):
+        """Creates a formatted string representing a temperature for the given
```

Comment:
The returned value is not necessarily a temperature. Please fix the description.

I am a bit confused by the term "channel" when the sensor returns 2-3 values as a single string. Are you treating each reading as a separate channel in the higher-level code?

Reply:
Good points. I modified the doc string and gave the variable a better name.